includes/custom-templates.md

```diff
@@ -0,0 +1,297 @@
+Templates are passed as options that end with **...Template** (in jQuery, Angular, and Vue) or **...Render**/**...Component** (in React).
+
+Each template has access to the following parameters:
+
+- `data`     
+A data source object or an object with fields specific for a particular template. For information on the contents of `data`, refer the template's API reference article and the **Template Data** section in it.
```

Comment:
Do you mean:

`A data source object or an object with fields specific to a particular template. For information on the contents of `data`, refer to the **Template Data** section of the template's API reference article.`